ddp-workspace/projects/ddp-brain/src/app/app.module.ts

```diff
@@ -84,6 +89,9 @@ sdkConfig.projectGAToken = DDP_ENV.projectGAToken;
 sdkConfig.errorReportingApiKey = DDP_ENV.errorReportingApiKey;
 sdkConfig.projectGcpId = DDP_ENV.projectGcpId;
 sdkConfig.doGcpErrorReporting = DDP_ENV.doGcpErrorReporting;
+sdkConfig.prismColumns = ['guid', 'shortId', 'userName', 'email', 'enrollmentStatus', 'dashboardLink', 'invitationId', 'proxyGuid', 'proxyShortId', 'proxyUserName'];
```

Comment:
These columns are supposed to be different for different studies, aren't they ?

Reply:
yes! and they are different. For example angio didn't have invitationId column